vamdctap-webservice/src/main/java/org/vamdc/tapservice/api/TAPInterface.java

```diff
@@ -19,6 +21,7 @@
 	@GET
 	@Produces({"application/x-xsams+xml","application/x-votable+xml","text/xml"})
 	public abstract Object getSyncResponse(
+			@Context HttpServletRequest requestHttp,
```

Comment:
I'm still wondering if we could avoid passing the entire request to those methods.